content/blog/supervisory-authorities.fr.md

```diff
@@ -0,0 +1,62 @@
+{
+    "title": "Autorités de contrôle de la protection des données",
+    "type": "autorites-de-controle",
+    "url": "autorites-de-controle",
+    "aliases": [
+    	"supervisory-data-protection-authorities",
+    	"dpas",
+    	"svas"
+    ],
+    "date": "2019-02-07T16:58:19+02:00",
+    "description": "Les autorités de contrôle de la protection des données sont des structures indépendantes dont la tâche est de veiller au respect des lois sur la protection des données. Si tu penses qu'une entreprise enfreint tes droits, tu peux les contacter gratuitement. Nous t'expliquons ici quelles sont les autorités de contrôle et t'aidons à en trouver une à contacter si tu veux déposer une plainte.",
+    "featured_image": "/card-icons/authority.svg",
+    "tags": [ "plainte", "compétence" ],
+    "authors": [ "baltpeter" ],
+    "blog_like": true
+}
+
+<img class="offset-image offset-image-left" src="/card-icons/authority.svg" style="height: 200px;">
+
+Les autorités de contrôle de la protection des données sont une institution importante pour la protection des données. En tant que structures indépendantes, elles ont pour mission de veiller au respect des lois sur la protection des données, en particulier le RGPD.
+
+Un certain nombre de pouvoirs leur sont conférés à cet effet. Tout d'abord, les autorités de contrôle disposent de pouvoirs de contrôle qui leur permettent de vérifier si un responsable de traitement respecte le droit applicable en matière de protection des données. Elles peuvent, par exemple, demander la liste des activités de traitement et d'autres informations dont elles ont besoin pour leurs tâches et même avoir accès aux locaux de l'entreprise (voir l'article 58, paragraphe 1, du RGPD).  
+Si l'autorité constate une faute du responsable du traitement, elle lui ordonnera d'abord de cesser cette activité et de se conformer à la loi. En fonction de la gravité de l'infraction, elle peut également imposer des amendes si nécessaire.
+
+Pour toi, en tant que consommateur, les autorités de contrôle offrent avant tout une aide directe si tu veux te défendre contre une entreprise ou une autre organisation qui ne traite pas correctement tes données. Si tu estimes qu'un responsable du traitement des données viole ta vie privée, tu peux à tout moment contacter les autorités de contrôle et leur demander de répondre à tes préoccupations.  
+Cette démarche est beaucoup plus facile pour toi que l'alternative consistant à intenter une action en justice contre le responsable du traitement. Le travail de l'autorité est même gratuit pour toi (voir l'article 57, paragraphe 3, du RGPD).
+
+<a id="one-stop-shop"></a>
+## Le principe du guichet unique et la compétence des autorités de contrôle
```

Comment:
I'm certainly not qualified to judge this but at least Linguee, a site I like to use for these cases as it references existing translations, including EU law, seems to agree with your translation of 'one-stop shop': https://www.linguee.com/english-french/translation/one+stop+shop.html

Reply:
That's already how I translated it, but it sounds strange in French. Is that an expression? Maybe we can propose something else which will fit better 😄

Reply:
In fact I already use DeepL and Linguee to help me making translations.

Reply:
Is "One-Stop Shop" used to define the fact that we contact only one supervisory authority which take care of everything instead of making procedures everywhere?

Reply:
> In fact I already use DeepL and Linguee to help me making translations.

Oh yeah, DeepL is great and very helpful, too.

> Is "One-Stop Shop" used to define the fact that we contact only one supervisory authority which take care of everything instead of making procedures everywhere?

Exactly, there is only a single SVA responsible for you and they are your single point of contact. This is in contrast to the previous model where you had to contact many different SVAs in the different countries depending on where the company you're complaining against was based.

Reply:
I'm gonna think about it in order to find a better translation. Litteral translation is bad in French.

Reply:
Well finally I made some research and apparently it's okay to translate it as "guichet unique" even if it sounds a bit weird.